src/context/UserContext.js

```diff
@@ -1,15 +1,25 @@
 import { createContext, useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
+import { hostUrl } from '../common/urls';
 
 export const UserContext = createContext();
 
 export const UserProvider = ({ children }) => {
     const [user, setUser] = useState(null);
+    const [userInfo, setUserInfo] = useState({});
```

Comment:
I will approve that for now, but this code should not be here.

Reply:
Where better to be? I tried to save userInfo here.